spring-boot-project/spring-boot-tools/spring-boot-maven-plugin/src/test/resources/pom_examples/jvm-args-1.xml

```diff
@@ -0,0 +1,39 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
We have integration tests for this. This sample project should move to `src/test/it`

Reply:
You mean `spring-boot-maven-plugin/src/it`?

Reply:
Yes